apps/dot-voting/app/components/VoteDetails/CastVote/EditVoteOption.js

```diff
@@ -68,7 +69,7 @@ const EditVoteOption = ({
           onUpdate={x => onUpdate(Math.round(x * 100))}
           value={value / 100}
         />
-        <HiddenLabel htmlFor="percentage">Percentage</HiddenLabel>
+        <HiddenLabel htmlFor={'percentage' + index}>Percentage {index}</HiddenLabel>
```

Comment:
I think it seems confusing, not helpful, to have the index show up in the user-facing text. The page will read: 

> 0x123 Percentage 1: 67%
> 0x456 Percentage 2: 33%

But it's not the first or second percentage for the label before it – it's the only percentage.

We only need the index here to tie the label to the input, so that clicking on the label focuses the input, and so screen readers understand how the two are related. In that case, we don't need something that's user-facing, and we can use `label`, which we already know is unique.

```jsx
<HiddenLabel htmlFor={`percentage-${label}`}>...
<TextInput id={`percentage-${label}`} ...

Reply:
Fixed!